### PR TITLE
minor fixes

### PR DESCRIPTION
**Pull request by ykris45**

### CODE REVIEW

File: data/i18n/en.yaml [hunk 1]
```diff
@@ -236,7 +236,7 @@ social:
   humanstext: Join the LBRY community.
   robotsheader: Robots
   robotstext: Make with us. All LBRY code is open source.
-  slack: Slack (chat)
+  discord: Discord (chat)
```
Comment (kauffj) [inline data/i18n/en.yaml]: The left side of these declarations is the input to an i18n call, e.g. `__('social.slack'`)`. I think in this case this one isn't used, but generally you can't change the key declaration without also changing the i18n calls themselves.

Comment (ykris45) [reply data/i18n/en.yaml]: whether this can be changed. instead of now migrating to discord. sir

**Comment by ykris45** [comment]
@kauffj can i change the key on https://github.com/lbryio/lbry.io/blob/0b4fa4368341772af32f40e78fd3cffcd4458510/controller/Controller.class.php line 87 to 'discord'  sir . hehe

**Comment by tzarebczan** [comment]
@ykris45 that's something completely different. I'm closing this, thanks.

**Pull request closed without merging**